### PR TITLE
Make integration tests more reliable

### PR DESCRIPTION
**Pull request by bishboria**
Attempt to improve the reliability of two integration tests (`campaign_edit_test` and `video_edition_create_edit_test`) as they fail regularly on CI.

These changes seem to make the test suite a lot more stable - Famous Last Words.

### CODE REVIEW

File: test/integration_test_helper.rb [hunk 1]
```diff
@@ -194,9 +195,22 @@ def save_edition_and_assert_success
...
+                      page.has_selector?("input[type=submit]#save-edition:enabled"))
+    assert save_attempted, "Failed to attempt saving the edition"
+
+    saved = (page.has_content?("edition was successfully updated.") ||
+             page.has_css?('.workflow-message', text: 'Saved'))
+    assert saved, "Failed to save the edition"
+  end
+
+  def save_edition_and_assert_success_without_ajax
+    with_javascript = false
```
Comment (boffbowsh) [inline test/integration_test_helper.rb]: I think it'd be more readable without this variable

**Comment by boffbowsh** [comment]
Tiny comment, but otherwise heroic effort :clap:

File: test/integration_test_helper.rb (hunk 1 above)
Comment (rboulton) [inline test/integration_test_helper.rb]: Wouldn't that make it just say `save_edition(false)` on the next line?  I think that would be quite hard to read.

**Comment by rboulton** [comment]
I'm going to merge this, since I really want master to have a successful build.  Feel free to discuss better ways to make the "save_edition(...)" bit readable though, and follow up in a subsequent PR if needed.

**Comment by fofr** [comment]
A little late to this, but I'm not sure what `save_edition_and_assert_success_slow` means and when to use it.

**Pull request merged**